gamemaker.py

```diff
@@ -1,98 +1,95 @@
+        lm = Landmark(name=name, clue=clue, question=question, answer=answer)
+        lm.save()
 
         return "Landmark " + name + " has been added!"
 
     def display_landmarks(self):
-        landmarks = self.database.get_landmarks()
+        landmarks = Landmark.objects.all()
         ret = ""
 
-        if len(landmarks) == 0:
+        if len(landmarks) == 1:
             ret = "There are no landmarks"
 
         else:
             for landmark in landmarks:
-                ret += landmark.get_name() + "\n"
+                if landmark.name != "dummy":
+                    ret += landmark.name + "\n"
 
         return ret
 
     def remove_landmark(self, input):
-        landmarks = self.database.get_landmarks()
         found = False
+        name = input[0]
 
-        for landmark in landmarks:
-            if landmark.get_name() == input[0]:
-                if self.database.remove_landmark(landmark):
-                    found = True
+        try:
+            landmark = Landmark.objects.get(name=name)
+            landmark.delete()
+            found = True
+        except Landmark.DoesNotExist:
+            pass
 
         if found:
-            return "Removed " + input[0] + " from landmarks."
+            return "Removed " + name + " from landmarks."
         else:
-            return "Couldn't find landmark with name " + input[0]
+            return "Couldn't find landmark with name " + name
 
     def display_status(self):
         string = ''
-        teams = self.database.get_teams()
+        teams = HuntUser.objects.all()
         for team in teams:
-            string += team.username + '\n' #+ ' is at ' + team.landmark + ' with ' + team.penalties + ' penalties\n'
+            if team.name != "maker":
+                string += team.name + '\n' #+ ' is at ' + team.landmark + ' with ' + team.penalties + ' penalties\n'
         if string == '':
             string = 'No teams!'
         return string
-        
-    def logout(self):
-        return_value = False
-        current_user = self.database.get_current_user()
-        if current_user is self:
-            self.database.set_current_user(None)
-            return_value = True
-        return return_value
 
     def display_menu(self):
-        return "Options\n\ndisplaystatus\nmaketeam [team name] [team password]\neditteam [team name to edit] [new team name] [new team password]\n" \
-        "deleteteam [team name to delete]]\n addlandmark [name], [clue], [question], [answer]\ndisplaylandmarks\nremovelandmark [name]\nlogout\n"
+        return "Options\n\ndisplaystatus\nmaketeam [team name], [team password]\neditteam [team name to edit], [new team name], [new team password]\n" \
+        "addlandmark [name], [clue], [question], [answer]\ndisplaylandmarks\nremovelandmark [name]\nsetpenalties [new time penalty], [new guess penalty]\n" \
+        "creategame [landmark name]...\nstartgame\nendgame\nlogout\n"
```

Comment:
Bit confusing from the users point of view on the syntax of create game. Do we separate the landmark names by spaces? Periods? Commas? Do we only put one landmark name? What does ... mean?

Reply:
I'll change it to have the following:
creategame [landmark name 1], [landmark name 2]...
that should be clear enough for a command line interface